jxls/src/main/java/org/jxls/util/CellRefUtil.java

```diff
@@ -236,14 +240,15 @@ static boolean nameLooksLikeBooleanLiteral(String rawSheetName) {
         switch (rawSheetName.charAt(0)) {
             case 'T':
             case 't':
-                return "TRUE".equalsIgnoreCase(rawSheetName);
+                return isaBooleanT(rawSheetName);
```

Comment:
I don't find isaBooleanT that understandable. The old code was easier to understand.